src/main/java/io/reactivex/internal/operators/observable/ObservableReplay.java

```diff
@@ -751,6 +757,26 @@ void truncateFinal() {
                 }
             }
         }
+        /* test */ final void collectObjectsInMemory(Collection<? super T> output) {
```

Comment:
There is a `collect(Collection)` method just above this addition, what's wrong with that?